CHANGELOG.md

```diff
@@ -1 +1,30 @@
+# 0.6.0
+
+## Prelude
+
+Added new features and commands, such as:
+
+- 'subgit delete'
+- 'subgit import'
+- 'subgit reset'
+- 'subgit clean'
+
+## General fixes
+
+- Added error handling for 'subgit pull'.
+	- If subgit config file has a repo with `branch: `, the command exits
+
+- Implemented multiprocessing for 'subgit fetch' commnand. [#41][https://github.com/dynamist/subgit/pull/41]
+
+## New features
+
+* [#42][https://github.com/dynamist/subgit/pull/42] - Added function to render all repos from a github/gitlab user account/organisation
```

Comment:
The word `render` feels wrong context or purpose here. Try to find some other wording for this if possible.